src/uct/tcp/tcp_cm.c

```diff
@@ -60,16 +60,20 @@ void uct_tcp_cm_change_conn_state(uct_tcp_ep_t *ep,
                                str_remote_addr, UCS_SOCKADDR_STRING_LEN));
 }
 
-static void uct_tcp_cm_io_err_handler_cb(void *arg, int errno)
+static ucs_status_t
+uct_tcp_cm_io_err_handler_cb(void *arg, int err_no)
```

Comment:
minor: better used `errno` here and in `uct_tcp_ep_io_err_handler_cb`, because it is well known/used name

Reply:
fixed